DigitalLearningSolutions.Web/Views/ResetPassword/Error.cshtml

```diff
@@ -0,0 +1,9 @@
+@using DigitalLearningSolutions.Web.ViewModels.Common
+<h1 class="nhsuk-heading-xl">Something went wrong...</h1>
+<p class="nhsuk-body-m">
+  There was a problem resetting your password.
+  The reset password link is only valid once, and expires after 2 hours.
+  Please try again.
+</p>
+
+<partial name="_ActionLink" model="@(new ActionLinkViewModel("ForgotPassword", "Index", "Reset password"))" />

```

Comment:
Hmm.. I'm wondering whether it's worth making this VC so we can use it with a similar syntax as back button (I don't think we'd ever create the View model in the code so might make sense here) - what do you think?

Reply:
Sounds fine to me! I'll do that.

Reply:
Done.